packages/react-components/react-motion/library/src/index.ts

```diff
@@ -20,3 +20,5 @@ export type {
   PresenceDirection,
   MotionImperativeRef,
 } from './types';
+
+export { useMotionDisableContext, MotionDisableProvider } from "./contexts/MotionDisableContext";
```

Comment:
```suggestion
export { MotionDisableProvider } from "./contexts/MotionDisableContext";
```

Let's do not export consumer now